test/test.env

```diff
@@ -1 +1 @@
-LOG_LEVEL=error
+LOG_LEVEL=warn
```

Comment:
So, to double check, we are happy keeping this as 'warn', not reverting back to 'error'?

Reply:
changed to warn to include a bit more details, than just error for a bit more descriptive output in github checks, changed back

Reply:
cool :)